### ISSUE TITLE
关于数据集

### ISSUE THREAD

**Issue by ghost** [chenchen235/SemStereo#1]
您好，关于US3D数据集加载，您TXT中的路径内容是重新做了分类整理吗？比如源数据似乎并没有左右的文件夹分类，且验证集数据也不一致。感谢解答！

**Comment by chenchen235** [comment]
1. The official dataset has distinguished between left and right, but I don't quite remember the details. It may require necessary preprocessing.

2. The download channel for the ground truth labels of the official validation set has either not been disclosed or expired long ago. Only the training set has labels, so the validation set cannot be used as a metric for verification. This is also why previous works, such as HMSMNet and DSM-Net, re-split the training set on their own. However, it is a pity that they did not disclose their specific splitting criteria. Therefore, we had to re-split the dataset and re-implement all open-source or easily reproducible models under this unified splitting standard for comparison. We also standardized all training configurations to ensure fairness as much as possible and made our specific splitting criteria publicly available for reference in future work.
